WORKSPACE

```diff
@@ -50,8 +50,12 @@ maven_install(
         "https://jcenter.bintray.com/",
         "https://repo1.maven.org/maven2",
     ],
+    maven_install_json = "//:maven_install.json",
```

Comment:
Can you add a comment with instructions to update this?

Reply:
Done.